src/io/files.js

```diff
@@ -1331,15 +1379,29 @@ p5.PrintWriter = function(filename, extension) {
    * Clears the data already written to the PrintWriter object
    * @method clear
    * @example
-   * <div class ="norender notest"><code>
-   * // create writer object
-   * let writer = createWriter('newFile.txt');
-   * writer.write(['clear me']);
-   * // clear writer object here
-   * writer.clear();
-   * // close writer
-   * writer.close();
-   * </code></div>
+   * <div>
+   * <code>
+   * function setup() {
+   *    createCanvas(100, 100);
+   *    background(234, 40, 95);
+   *    button = createButton('CLEAR ME');
+   *    button.style('font-size', '8px');
+   *    button.position(21, 40);
+   *    button.size(60, 20);
+   *    button.mousePressed(createFile);
+   * }
+   * function createFile() {
+   *    // create writer object
+   *    let writer = createWriter('newFile.txt');
+   *    writer.write(['clear me']);
+   *    // clear writer object here
+   *    writer.clear();
+   *    // close writer
+   *    writer.close();
+   *    // file will be empty
+   * }
+   * </code>
+   * </div>
```

Comment:
it's great to have a complete example here! but can you add it as a second example and keep the original as well?